test/fixtures/dialects/ansi/with_insert_statement_b.yml

```diff
@@ -0,0 +1,57 @@
+# YML test files are auto-generated from SQL files and should not be edited by
+# hand. To help enforce this, the "hash" field in the file must match a hash
+# computed by SQLFluff when running the tests. Please run
+# `python test/generate_parse_fixture_yml.py`  to generate them after adding or
+# altering SQL files.
+_hash: 4e55be927c4b974020c6f1ad74a4788a3944c840e8ac30e8d9a64c342d9c2e5d
+file:
+  statement:
+    with_compound_statement:
+      unparsable:
+      - keyword: WITH
+      - raw: mycte
```

Comment:
I think the behaviour you're seeing here will be resolved by the merge of #1894

Reply:
@jpy-git the fix has been merged now - so pulling in main may make this problem go away

Reply:
@WittierDinosaur amazing thank you, I'll take a look now!